records_mover/session.py

```diff
@@ -194,6 +221,42 @@ def _boto3_session(self) -> Optional['boto3.session.Session']:
         else:
             return self.creds.boto3_session(self._default_aws_creds_name)
 
+    def _gcs_creds(self) -> Optional['google.auth.credentials.Credentials']:
+        # TODO: Should save this
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker